packages/smooth_app/lib/data_models/continuous_scan_model.dart

```diff
@@ -228,4 +233,11 @@ class ContinuousScanModel with ChangeNotifier {
     await _refresh();
     notifyListeners();
   }
+
+  set lastConsultedBarcode(String? barcode) {
```

Comment:
Please put the code of the setter at the same position as the getter.

Reply:
> Please put the code of the setter at the same position as the getter.

✅